### PR TITLE
Portfolio10

### PR DESCRIPTION
**Pull request by izzybaer**
added .map and .reduce and an IIFE

### CODE REVIEW

File: scripts/app.js [hunk 1]
```diff
@@ -1,62 +1,60 @@
...
 
-var images = [];
+var app = app || {};
 
-function Project (rawDataObj) {
-  for (var key in rawDataObj) {
-    this[key] = rawDataObj[key];
+(function(module) {
+  function Project (rawDataObj) {
+    for (var key in rawDataObj) {
```
Comment (Tenlia) [inline scripts/app.js:7]: is there a way you could replace this for loop?

File: scripts/app.js [hunk 2]
```diff
@@ -1,62 +1,60 @@
...
+    var templateRender = Handlebars.compile(template);
+    return templateRender(this);
+  };
+
+  Project.loadAll = function(rawData) {
+    Project.all = rawData.map(function(project) {
+      return new Project(project);
+    })
+
+    Project.all.forEach(function(project) {
```
Comment (Tenlia) [inline scripts/app.js:25]: this for loop should be replaced

File: scripts/app.js [hunk 3]
```diff
@@ -1,62 +1,60 @@
...
+      $('#portfolio').append(project.toHtml());
     });
+
+    console.table(Project.all);
```
Comment (Tenlia) [inline scripts/app.js:29]: no console logs (I know it's a table, it counts!) in assignment submissions

File: scripts/app.js [hunk 4]
```diff
@@ -1,62 +1,60 @@
...
+      function(err) {
+        console.log(err);
+      });
+    }
   }
 
-});
 
-//This way easier to show the tab content, compared to using the "data-content"
+  // $(document).ready(function(){
```
Comment (Tenlia) [inline scripts/app.js:55]: dead code

File: scripts/app.js (hunk 1 above)
Comment (izzybaer) [reply scripts/app.js:7]: Dear Liz,

I don't know if there is a way I could replace this for loop. I don't think so. Not today, satan!

Love, Izzy

File: scripts/app.js (hunk 2 above)
Comment (izzybaer) [reply scripts/app.js:25]: okay, fine. This time Liz....

File: scripts/app.js (hunk 3 above)
Comment (izzybaer) [reply scripts/app.js:29]: does. NOT COUNT.

File: scripts/app.js (hunk 4 above)
Comment (izzybaer) [reply scripts/app.js:55]: THAT WAS ALIVE

**Pull request merged**